ui/lib/pki/addon/templates/roles/index.hbs

```diff
@@ -16,50 +16,58 @@
   </ToolbarActions>
 </Toolbar>
 
-{{#if this.model.roleModel.length}}
-  {{#each this.model.roleModel as |pkiRole|}}
-    <LinkedBlock class="list-item-row" @params={{array "roles.role.details" pkiRole.id}} @linkPrefix={{this.mountPoint}}>
-      <div class="level is-mobile">
-        <div class="level-left">
-          <div>
-            <Icon @name="user" class="has-text-grey-light" />
-            <span class="has-text-weight-semibold is-underline">
-              {{pkiRole.id}}
-            </span>
+{{#if this.model.hasConfig}}
+  {{#if this.model.roleModel.length}}
```

Comment:
nit: the name `roleModel` makes it sound like we expect this to be a singular model, when really it's the list of roles. Same with `certificateModel` naming. Personally I'd prefer something more indicative of what the item is, like `roles` or `roleList`.